src/main/scala/com/nec/colvector/BytePointerColVector.scala

```diff
@@ -92,20 +89,78 @@ final case class BytePointerColVector private[colvector] (
       val handles = (Seq(structPtr) ++ buffers).zipWithIndex.map{case (ptr, idx) =>
         process.putAsync(ptr, veMemoryPositions(idx))
       }
-      VeAsyncResult(handles){ () =>
+      OldVeAsyncResult(handles){ () =>
```

Comment:
Not a fan of calling things old.  Prefer to leave as is and name the new things as "New".  Less changes.

Reply:
Updated